### PR TITLE
Remove buttons and related javascript, update styling for browse near…

### PR DESCRIPTION
**Pull request by aeschylus**
…by to enable scrollbar navigation

### CODE REVIEW

File: app/assets/javascripts/embedded-call-number-browse.js [hunk 1]
```diff
@@ -138,48 +138,25 @@
...
       // Scroll to the current document
-      function scrollOver(additional){
+      function scrollOver(){
         var current = $galleryDoc.docById();
-        var left;
+        var scrollAmount;
         if (current.length){
-          left = current.position().left;
+          console.log('I have entered the branch');
+          console.log(current.position());
```
Comment (jkeck) [inline app/assets/javascripts/embedded-call-number-browse.js]: Leftover debugging cruft?

File: app/assets/javascripts/embedded-call-number-browse.js [hunk 2]
```diff
@@ -138,48 +138,25 @@
...
+          scrollAmount = current.position().left - current.parent().width()/2 + current.innerWidth()/2;
         }else{
           console.log('could not get current position');
```
Comment (jkeck) [inline app/assets/javascripts/embedded-call-number-browse.js]: More debugging

**Comment by jkeck** [comment]
@aeschylus I think you mentioned something about adding a commit in w/ this PR to address the book-cover alignment issues in the embedded call number browse.

Perhaps either I was confused about where that was coming from, or it got bumped from these changes.  It looks like they're still not center-aligned.  Would you like me to take care of that in a separate PR?

**Comment by aeschylus** [comment]
Yep, I put this up since I was not able to run the tests locally to see if anything failed. This is not ready for merge.

**Comment by aeschylus** [comment]
@jkeck Looks like I committed the branch for that but never made a respective pr. 

https://github.com/sul-dlss/SearchWorks/pull/1238

Also removed the debug code from the JS here.

**Pull request merged**